client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -168,6 +210,34 @@ class GoogleAnalyticsForm extends Component {
 									{ translate( 'Where can I find my Tracking ID?' ) }
 								</ExternalLink>
 							</fieldset>
+							{ showAnonymizeIP &&
+								<fieldset>
+									<CompactFormToggle
+										checked={ fields.wga ? fields.wga.anonymize_ip : false }
```

Comment:
We should make sure to cast this as a bool before passing. Enabling it shows the correct status, but I get the following console warning:

> Warning: Failed prop type: Invalid prop `checked` of type `string` supplied to `FormToggle`, expected `boolean`.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17985/commits/51f29ed52a49f45dc7fd8b45a560a5e05acc8a39